.github/workflows/integration-tests.yml

```diff
@@ -688,7 +689,7 @@ jobs:
         uses: actions/github-script@v7
         with:
           script: |
-            core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
+            core.exportVariable('ACTIONS_RESULTS_URL', process.env.ACTIONS_RESULTS_URL || '');
```

Comment:
Also need to update lines 258, 542, and 602.

Reply:
i was sure i did a full replace :( 
Thanks!